.gitignore

```diff
@@ -8,4 +8,7 @@
 release
 .jekyll-cache
 .jekyll-metadata
-NOTES
\ No newline at end of file
+NOTES
+
+.direnv/
+result/
```

Comment:
`.direnv` seems fine but `result` might be a naming that's quite generic (could conflict or have lack of context). Is the naming always `result`?

Reply:
Could be a custom name though? `nix-result` maybe?

Reply:
it's always `result` AFAIK

Reply:
`nix build` ships target to `result/bin/`. I can confirm

Reply:
coolio, thanks for explaining. Would you ming leaving a comment that's a folder produced by `nix build`

Reply:
aye aye!